js-core/lesson2.md

```diff
@@ -153,11 +153,30 @@ countries.push('United States of America', 'France');
 
 4. **Research:**
 
-In the next class, we expect to do a short presentation about one of these topics:
+Read the chapter from JS: The Good parts
+- Chapter 9: Style - it's a short chapter, everyone start by reading that chapter.
+- ALL: Read the chapter about **Objects** (up to Reference)
+- Read the chapter about **Arrays** (ignore the Methods section)
+- Read the chapter about Functions up to the section about **Return**
+
+When reading always try the code snippets by yourself in Node. (change document.writeln to console.log)
+
+Next class - we want you to do a presentation (including showing some code sample) for one of these concepts:
+
+- Switch Statements and Ternary Operators
+- Math methods in JavaScript
+- While loops (compare them to for Loops)
+- Array.slice and Array.splice
+- Array.map and Array.foreach
+- Array.find and Array.filter
+- String.replace, String.search and other String methods
+- for ... in (How to loop the properties of an object)
```

Comment:
That's an ES6 feature. Do we want to cover this? Or maybe have them look into `Object.keys(myObj)`?